mailviews/tests/urls.py

```diff
@@ -1,13 +1,25 @@
-try:
-    from django.conf.urls import include, patterns, url
-except ImportError:
-    from django.conf.urls.defaults import include, patterns, url
+from mailviews.utils import is_django_version_greater_than_1_9
+
+if is_django_version_greater_than_1_9():
+    from django.conf.urls import include, url
+else:
+    try:
+        from django.conf.urls import patterns, include, url
+    except ImportError:
+        # Django <1.4 compat
```

Comment:
At this point I don't think Django Mailviews supports Django 1.3 or below, so we should be safe removing it.